core/src/main/scala/cromwell/core/io/AsyncIo.scala

```diff
@@ -1,84 +1,61 @@
 package cromwell.core.io
 
-import akka.actor.{Actor, ActorLogging, ActorRef}
+import akka.actor.ActorRef
+import cromwell.core.io.IoPromiseProxyActor.IoCommandWithPromise
 import cromwell.core.path.BetterFileMethods.OpenOptions
 import cromwell.core.path.Path
 
+import scala.concurrent.Future
 import scala.concurrent.duration._
-import scala.concurrent.{Future, Promise}
-import scala.language.postfixOps
 
+object AsyncIo {
+  val ioTimeout = 3.minutes
```

Comment:
I vaguely remember someone saying "if someone complains we'll make a knob" but I'm not sure. I'm sure @geoffjentry will have an opinion on this. I'm happy to add a knob in case someone wants it though.

Reply:
I caught myself in a contradiction lately where I was talking up all of the configuration options.

I guess the guiding principle should be "does it seem reasonable that someone is going to want to fiddle with this" instead of simply "OMG, it's a hardcoded value! knob it!"